### PR TITLE
Update credits / license attribution

### PR DESCRIPTION
**Pull request by Octobob**
An automated update of the credits / license attribution file, based on the latest code in https://github.com/OctopusDeploy/OctopusDeploy. 

Please review for
* syntactical correctness (ie, markdown looks correct)
* logical correctness (ie, when reading it, it makes sense)
* license conformance (we agree with the license in question and we are able to redistribute it)

Created by TeamCity project 'Automated Documentation Updates::Update credits.md (OSS license attribution file)', build 155

### CODE REVIEW

File: docs/credits.md [hunk 1]
```diff
@@ -912,6 +926,7 @@ Octopus Deploy is made possible thanks to the following great third-party produc
...
 | Octodiff                                          | Octopus Deploy                                                            | [https://github.com/OctopusDeploy/Octodiff](https://github.com/OctopusDeploy/Octodiff)                                                                                                                                      | [Apache-2.0](https://github.com/OctopusDeploy/Octodiff/blob/master/LICENSE.txt)                                     |
+| Octopus.Build.ConsolidateCalamariPackagesTask     | Octopus.Build.ConsolidateCalamariPackagesTask                             | []()                                                                                                                                                                                                                        | [Undefined]()                                                                                                       |
```
[the remark this answers is not in the document]
Comment (droyad) [reply docs/credits.md:929]: Is this what is needed? https://github.com/OctopusDeploy/ConsolidateCalamariPackagesTask/commit/a57621b758fb29bb59d4194f723115cf8704a3ee

[the remark this answers is not in the document]
Comment (matt-richardson) [reply docs/credits.md:929]: @andrew-moore-octo is probably the best bet to confirm that

[the remark this answers is not in the document]
Comment (andrew-moore-octo) [reply docs/credits.md:929]: Yep, a license URL should be sufficient. License names can be derived from that.

File: docs/credits.md [hunk 2]
```diff
@@ -278,16 +286,17 @@ Octopus Deploy is made possible thanks to the following great third-party produc
...
 | babel-plugin-jest-hoist                           |                                                                           | [https://github.com/facebook/jest](https://github.com/facebook/jest)                                                                                                                                                        | [MIT](https://spdx.org/licenses/MIT.html)                                                                           |
+| babel-preset-current-node-syntax                  | Nicol├▓ Ribaudo                                                           | [https://github.com/nicolo-ribaudo/babel-preset-current-node-syntax](https://github.com/nicolo-ribaudo/babel-preset-current-node-syntax)                                                                                    | [MIT](https://spdx.org/licenses/MIT.html)                                                                           |
```
[the remark this answers is not in the document]
Comment (andrew-moore-octo) [reply docs/credits.md:299]: Fixed in https://github.com/OctopusDeploy/DependencyScanner/pull/21

**Comment by andrew-moore-octo** [comment]
Closing. https://github.com/OctopusDeploy/docs/pull/734 has fixes for the character encoding issues in author names.

**Pull request closed without merging**